core/ir/x86/decode_table.c

```diff
@@ -5821,6 +5828,34 @@ const instr_info_t prefix_extensions[][12] = {
     {INVALID,    0xf30fae34, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
     {INVALID,    0x660fae34, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
     {INVALID,    0xf20fae34, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+  }, { /* prefix extension 189 */
+    {INVALID,      0x385218, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,    0xf3385218, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,    0x66385218, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,    0xf2385218, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    /* vex */
+    {INVALID,      0x385218, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {INVALID,    0xf3385218, "(bad)",   xx, xx, xx, xx, xx, no, x, NA},
+    {VEX_W_EXT,    0x385218, "(vex_W ext 112)",   xx, xx, xx, xx, xx, no, x, 112},
```

Comment:
It doesn't affect operation of the table but this really should be 0x66385218

Reply:
See https://github.com/DynamoRIO/dynamorio/issues/5483#issuecomment-1122239831 under "disassembler shows data16 prefix"

Reply:
Fixed the test and the entries.